config/rubocop_todo.yml

```diff
@@ -0,0 +1,14 @@
+# This configuration was generated by
+# `rubocop --auto-gen-config`
+# on 2016-02-18 19:14:30 +0100 using RuboCop version 0.37.2.
+# The point is for the user to remove these configuration records
+# one by one as the offenses are removed from the code base.
+# Note that changes in the inspected code, or installation of new
+# versions of RuboCop, may require this file to be generated again.
+
+# Offense count: 74
+# Cop supports --auto-correct.
+# Configuration parameters: EnforcedStyle, SupportedStyles.
+# SupportedStyles: when_needed, always
+Style/FrozenStringLiteralComment:
+  Enabled: false
```

Comment:
I think this should be enabled - it speeds up code significantly so maybe by default we should recommend this